lib/components/Admin/AdminSideNavigation.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Calendar,
+  Chart,
+  House,
+  SquareCaretLeft,
+  SquareCaretRight,
+  Table,
+  Users,
+  UsersRectangle,
+} from '@lib/icons';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useAtom } from 'jotai';
+import { sideNavToggleAtom } from '@lib/atoms';
+
+const links = [
+  {
+    href: '/admin',
+    text: 'Dashbord',
+    icon: <Table className="h-4 w-4" color="inherit" />,
+  },
+  {
+    href: '/admin/events',
+    text: 'Arrangementer',
+    icon: <Calendar className="h-4 w-4" color="inherit" />,
+  },
+  {
+    href: '/admin/users',
+    text: 'Brukere',
+    icon: <Users className="h-4 w-4" color="inherit" />,
+  },
+  {
+    href: '/admin/memberships',
+    text: 'Medlemskap',
+    icon: <UsersRectangle className="h-4 w-4" color="inherit" />,
+  },
+  {
+    href: '/admin/statistics',
+    text: 'Statistikk',
+    icon: <Chart className="h-4 w-4" color="inherit" />,
+  },
+];
+
+const AdminSideNavigation = () => {
```

Comment:
At side navigation utvider seg når man tar mouseover hadde vært kult, at den viser hele navnet på knappene. Future work maybe

Reply:
Yeh, vurdert egt. det først. Implementering hadde ikke vært vanskelig det heller. Tror en `hover:w-64` på className hadde funket fint liksom. Greia er bare at det er litt funky in practice, der noen få problemer kan oppstå. Type:
- Utvides litt fram og tilbake etter navigation mellom pages (dette er mulig å fikses på)
- Er ganske lett å hovere ved uhell, navigation vil da utvides hakkete litt fram og tilbake
- Ikon er ofte mye cleanere og forhåpentligvis selvforklarende (legge til en tooltip for bedre bruker vennlighet)

Tenkte derfor å løse dette ved å vise en tooltip box istedet. Se Figma f.eks.